packages/components/src/mobile/utils/alignments.native.js

```diff
@@ -3,12 +3,14 @@ export const WIDE_ALIGNMENTS = {
 		wide: 'wide',
 		full: 'full',
 	},
-	excludeBlocks: [ 'core/columns', 'core/heading' ],
+	innerContainers: [ 'core/group', 'core/columns', 'core/column' ],
```

Comment:
I think we should add a note somewhere that each new inner container needs to be listed here

Reply:
@dratwas Where I can leave this kind of message?

Reply:
+1 I'd say right above this line

Reply:
Done